pkg/registry/apis/provisioning/jobs/queue.go

```diff
@@ -97,10 +97,7 @@ func (s *jobStore) Add(ctx context.Context, job *provisioning.Job) (*provisionin
 	}, nil)
 
 	// For now, start a thread processing each job
-	go func() {
-		time.Sleep(time.Millisecond * 200)
-		s.drainPending()
-	}()
```

Comment:
I'm not sure if we need the sleep here? I removed it as it's already present in the for loop in the function called, and I'd rather avoid waiting as much as possible in our testing suite.

Reply:
I added it earlier so I would see the update more clearly 🤣